From 41f785bfb2b25830078534401a4cb82355cc54c1 Mon Sep 17 00:00:00 2001
From: Victor Wan <victor.wan@amlogic.com>
Date: Wed, 18 Jun 2014 15:28:38 +0800
Subject: [PATCH 4360/5965] uimage: change lzo compression level to 9

To fix below error:
   Image Name:   Linux-3.10.33
   Image Type:   ARM Linux Kernel Image (lzo compressed)
   Data Size:    7527994 Bytes = 7.2 MiB
   Load Address: 00008000
   Entry Point:  00008000
   Verifying Checksum ... OK
    Ramdisk start addr = 0x1272e800, len = 0x1e965a
    Flat device tree start addr = 0x12918000, len = 0x465c magic=0xedfe0dd0
   Uncompressing Kernel Image ... LZO: uncompress or overwrite error -6 - must RESET board to recover
resetting ...
---
 arch/arm/boot/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/Makefile b/arch/arm/boot/Makefile
index b0fc2fccb0ee..5547ee1247c0 100755
--- a/arch/arm/boot/Makefile
+++ b/arch/arm/boot/Makefile
@@ -30,7 +30,7 @@ karg="-z"
 else ifeq ($(CONFIG_KERNEL_LZO),y)
 UIMAGE_COMPRESSION ?= lzo
 kcomp=lzop
-karg=
+karg="-9"
 else ifeq ($(CONFIG_KERNEL_GZIP),y)
 UIMAGE_COMPRESSION ?= gzip
 kcomp=gzip
-- 
2.19.0

